tests/testthat/test-load-hooks.R

```diff
@@ -76,7 +76,7 @@ test_that("onLoad and onAttach", {
   # c: modified by onAttach in package env
   expect_equal(the$a, 2)
   expect_equal(the$b, 2)
-  expect_equal(the$c, 2)
+  expect_equal(the$c, 1)
```

Comment:
Because the package env is no longer populated by internal bindings at the time the user and package hooks are run. This is more consistent with normal package loading.

Reply:
Ok. You might need to update the comments too?